crystal_functions/file_readwrite.py

```diff
@@ -734,6 +734,226 @@ def get_config_analysis(self):
         self.atom_type2 = atom_type2
         return [self.atom_type1, self.atom_type2]
 
+    """
+    Thermodynamic-specific attributes, including:
+        self.edft: get_qpoint, DFT total energy at central point, with probable
+                   corrections. Unit: KJ / mol cell
+        self.nqpoint: get_qpoint, Number of q points
+        self.qpoint: get_qpoint, Fractional coordinates of qpoints
+        self.nmode: get_mode, Number of vibrational modes at all qpoints
+        self.frequency: get_mode, Frequencies of all modes at all qpoints.
+                        Unit: THz
+        self.eigenvector: get_eigenvector, Eigenvectors (classical amplitude)
+                          of all atoms all modes at all qpoints. Unit: Angstrom
+
+    By Spica. Vir., ICL. Jun. 01, 2022.
```

Comment:
Please remove names from the code. There is no need for this since GitHub tracks contributions line by line